src/trip.core.js

```diff
@@ -507,7 +507,7 @@ Trip.prototype = {
     this.setTripBlock(o);
     this.showTripBlock(o);
 
-    if (this.settings.enableAnimation) {
+    if (TripUtils.isSet(o.enableAnimation, this.settings.enableAnimation)) {
```

Comment:
I was wondering about the TripUtils.isSet function name too. In doubt, I went for the shortest. I do not like the call to an external function but did not want to clone the same functionality everywhere it was required. Maybe it should be more explicit about the local/global logic. I was also wondering if the tripObject should not have a function itself `o.settingApplies("enableAnimation")` which would resolve the setting using the local first, or otherwise use the global ones. It would be less prone to error which currently allows to send any value as a parameter (e.g. `isSet(o.settingA, this.settings.settingB)`.